src/utils/check_is_internal_function.ts

```diff
@@ -0,0 +1,8 @@
+import { FUNCTIONS_INTERNAL_DIR } from '../runtimes/constants.js'
+
+export const checkIsInternalFunction = (srcDir = '') => {
+  const BACKSLASH_REGEXP = /\\/g
+
+  // Backslashes need to be converted for Windows.
+  return srcDir?.replace(BACKSLASH_REGEXP, '/').includes(FUNCTIONS_INTERNAL_DIR)
```

Comment:
Can you please use `unixify()` instead of the custom regex.
Also we should use `is-path-inside` to check if the srcDir is inside `FUNCTIONS_INTERNAL_DIR`, as a path called `.netlify/internal-functions-not/` will happily match even though it is not our internal folder. `is-path-inside` takes care of that.

Reply:
In the current setup, `is-path-inside` doesn't seem to work because `FUNCTIONS_INTERNAL_DIR` is not a full path. I changed the regex into unixify, and I'll figure out the path inside logic once I pick up Eduardo's comment, because I think then I'll have a more thorough logic that will have a full path for the internal directory :)